docs/releases/v3.2.2.rst

```diff
@@ -0,0 +1,32 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Version 3.2.2
+=============
+
+*Released 2020-05-14*
+
+Invenio v3.2.2 fixes issues with incompatibilities with several libraries:
+- WTForms recently release has moved `email_validator` to an optional
+dependency. It has been added in the corresponding packages.
+- SQLAlchemy-Utils v0.36 breaks compatibility of Invenio models with MySQL. It
+enforces setting the length on type `VARCHAR`.
+- Sphinx v3.0+ breaks the build of documentation, due to a Flask context issue.
+- Flask is now provided as a centrally managed dependency from `Invenio-Base`.
```

Comment:
Is this not only in Invenio v3.3?

Reply:
Sadly no, many backports needed this changes to fix the build (Sphinx + Flask). SQLAlchemy-Utils is somehow the same, they release in between and "broke" our compatibility. So the pin needed to be backported (invenio-db v1.0.5, which is compatible with v3.2 and v.3.3)